doc/releases/index.rst

```diff
@@ -115,7 +114,20 @@ to be able to understand the context of the change.
    :glob:
    :reversed:
 
-   migration-guide-*
+   migration-guide-3.[6-7]
+   migration-guide-4.[0]
+
+End-of-life Releases
+********************
+
+.. toctree::
+   :hidden:
+   :maxdepth: 1
+
+   eol_releases
```

Comment:
Uhm, it seems that this effectively does nothing as no table of contents shows up: https://builds.zephyrproject.io/zephyr/pr/80859/docs/releases/index.html#end-of-life-releases

Reply:
it's basically only there to "mount" the document in the overall document hierarchy

Reply:
Ahh, my bad. I see in the document hierarchy there are two `End-of-life releases` levels. Could changing `:maxdepth:` to 2 make it so that `Release Notes` and `Migration Guides` appear in the hierarchy? Or rather maybe just remove the first title, [End-of-life releases](https://builds.zephyrproject.io/zephyr/pr/80859/docs/releases/eol_releases.html#end-of-life-releases), so that the subtitles appear directly in the hierarchy? Don't know. :thinking: